libraries/abstractions/common_io/test/iot_test_common_io.c

```diff
@@ -56,14 +56,6 @@ TEST_GROUP_RUNNER( Common_IO )
 {
     size_t i = 0;
 
-    #ifdef IOT_TEST_COMMON_IO_PERFCOUNTER_SUPPORTED
```

Comment:
This shouldn't get removed

Reply:
Oops. Merge mistake during rebase. Fixed in latest commit. Thanks for catching